tests/commands/test-kob-install.sh

```diff
@@ -1 +1,328 @@
+#!/bin/bash
+
+path_to_kob_envs="${KOBMAN_DIR}/envs"
+path_to_kob_env_tests=~/KOBman/tests/envs
+kobman_env_name=${1:-dummyenv}
+version=$2
+KOBMAN_NAMESPACE=${KOBMAN_NAMESPACE:-hyperledgerkochi}
+status="true"
+
+function __test_kob_run
+{
+  __test_kob_init
+  __test_kob_execute
+  __test_kob_validate || return 1
+  __test_kob_cleanup 
+
+}
+
+function __test_kob_init {
+  echo "checking for kob..."
+
+  if [[ -d ${KOBMAN_DIR} ]]; then
+
+    echo "kob found"
+    source "${KOBMAN_DIR}/bin/kobman-init.sh"
+  
+  else
+  
+    echo "kob not found"
+    echo "installing kobman...."
+    curl -L https://raw.githubusercontent.com/${KOBMAN_NAMESPACE}/KOBman/master/get.kobman.io | bash > /dev/null 2>&1
+    source "${KOBMAN_DIR}/bin/kobman-init.sh"
+  
+  fi
+  
+  echo "creating and sourcing dummyenv files..."
+  touch $KOBMAN_DIR/var/kobman_env_$kobman_env_name.proc
+  
+  __test_kob_insert_dummyenv_version
```

Comment:
__test_kob_insert_dummyenv_version,__test_kob_install_dummyenv_script 
 functions need not have __test as they will not be called in the run function or in any integration tests..